core/src/main/java/org/mule/runtime/core/internal/registry/MuleRegistryHelper.java

```diff
@@ -52,31 +53,31 @@
   /**
    * A reference to Mule's internal registry
    */
-  private Registry registry;
+  private final Registry registry;
```

Comment:
Also this one?

Reply:
it is being set in the constructor. The `final` is added automatically by the IDE when it detects that  it is not being written anywhere else.